CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Unreleased
 
+## ScrollingAnimation: Fixed issue where the animation could become out of sync if the properties on ScrollingAnimation were updated.
```

Comment:
We usually have a short heading, and a bullet-list under it. Why should this entry be different?